microcosm_flask/conventions/crud.py

```diff
@@ -60,7 +68,13 @@ def search(**path_data):
             result = definition.func(**merge_data(path_data, page.to_dict(func=identity)))
             response_data, headers = page.to_paginated_list(result, ns, Operation.Search)
             definition.header_func(headers)
-            return dump_response_data(paginated_list_schema, response_data, headers=headers)
+            response_format = self.negotiate_response_content(definition.response_formats)
```

Comment:
You probably also need to instrument the relation convention routes...

There's probably also a refactoring we could do to make the endpoint smarter so we do something more like: `endpoint.dump(...)` and then could make this kind of change in one place instead of many. I assume this is out of scope.